src/fontra/views/editor/panel-glyph-search.js

```diff
@@ -26,6 +26,7 @@ export default class GlyphSearchPanel extends Panel {
       glyphsSearch.updateGlyphNamesListContent();
     });
     this.editorController.fontController.ensureInitialized.then(() => {
+      this.editorController.fontController.glyphsSearch = glyphsSearch;
```

Comment:
I try to get `filteredGlyphItems` from `glyphsSearch`, without running it each time I use **next/previous glyph**.

Reply:
One idea:
- make `glyphsSearch` an attr of `GlyphSearchPanel` instead of it being a local var in the constructor
- then I think you should be able to access it from `EditorController`

The font controller has nothing to do with these kinds of things, and you should never assign random properties to it.

Reply:
I first replaced:
`const glyphsSearch = this.contentElement.querySelector("#glyphs-search");`
with
`this.glyphsSearch = this.contentElement.querySelector("#glyphs-search"); `
But I was not able to access `this.glyphsSearch` within EditorController. 
Why I made it this way: [c58bba6](https://github.com/googlefonts/fontra/pull/1706/commits/c58bba6b0bae7ebbf84fe376190b3d426ce67389)

But I think this is not what your vision was, am I right? 
I am sorry in advance.

Reply:
> But I was not able to access this.glyphsSearch within EditorController.

No, because it is then a property of the panel (which is good). 

In the EditorController you first have to find the panel (I think with `this.getSidebarPanel()`), and then you can access `panel.glyphsSearch`.

Reply:
Thank you soooo much for your guidance. I changed it and set the PR back to 'Ready for review'.